mgmtworker/mgmtworker/task_consumers/cluster_service_consumer.py

```diff
@@ -0,0 +1,66 @@
+########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+############
+
+import logging
+
+from cloudify.constants import CLUSTER_SERVICE_EXCHANGE_NAME
+from cloudify.amqp_client import TaskConsumer
+try:
+    from cloudify_premium.ha import syncthing
+except ImportError:
+    update_devices = None
+
+logger = logging.getLogger('worker.mgmtworker')
+
+
+class ClusterServiceConsumer(TaskConsumer):
+    """
+    This consumer is intended to be used only by the mgmt-worker and only for
```

Comment:
'mgmt-worker' we normally just write in one work mgmtworker

Reply:
Fixed